src/Symfony/Component/Console/Tests/Descriptor/ObjectsProvider.php

```diff
@@ -43,6 +43,8 @@ public static function getInputOptions()
             'input_option_4' => new InputOption('option_name', 'o', InputOption::VALUE_IS_ARRAY | InputOption::VALUE_OPTIONAL, 'option description', array()),
             'input_option_5' => new InputOption('option_name', 'o', InputOption::VALUE_REQUIRED, "multiline\noption description"),
             'input_option_6' => new InputOption('option_name', array('o', 'O'), InputOption::VALUE_REQUIRED, 'option with multiple shortcuts'),
+            'input_option_7' => new InputOption('option_name', 'o', InputOption::VALUE_REQUIRED, 'option description', '<comment>style</>'),
```

Comment:
We should IMO use more descriptive names. Adding a counter suffix leads to merge conflicts that could be avoided.